mythril/laser/ethereum/svm.py

```diff
@@ -480,6 +483,47 @@ def _new_node_state(
 
         new_node.function_name = environment.active_function_name
 
+    def register_hooks(self, hook_type: str, hook_dict: Dict[str, List[Callable]]):
+        """
+
+        :param hook_type:
+        :param hook_dict:
+        """
+        if hook_type == "pre":
+            entrypoint = self.pre_hooks
+        elif hook_type == "post":
+            entrypoint = self.post_hooks
+        else:
+            raise ValueError(
```

Comment:
The exceptions raised on a failure to find a hook are inconsistent. It's a `ValueError` in `register_hooks`, and a `NotImplementedError` for `register_laser_hooks`. Is this appropriate, or should we use the same exception for both?